src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -282,8 +293,8 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
     throw Error('App url No provided or it is invalid.')
   }
 
-  if (appTimeout || appLoadError) {
-    throw Error('There was an error loading the Safe App. There might be a problem with the app provider.')
+  if (appLoadError) {
+    throw Error('There was an error loading the Safe app. There might be a problem with the app provider.')
```

Comment:
The capitalization was intentional, I think. To differentiate from the Safe web app itself. This whole nomenclature is very confusing.